ligo/Makefile

```
@@ -39,6 +42,22 @@ ifeq ($(OPTIMIZE), true)
 	$(MORLEY) optimize --contract $(OUT)/baseDAO.tz --output $(OUT)/baseDAO.tz
 endif
 
+
+$(OUT)/registryDAO_storage.tz : admin_address = tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af
+$(OUT)/registryDAO_storage.tz : token_address = tz1QozfhaUW4wLnohDo6yiBUmh7cPCSXE9Af
+$(OUT)/registryDAO_storage.tz : a = 0n
+$(OUT)/registryDAO_storage.tz : b = 0n
+$(OUT)/registryDAO_storage.tz : s_max = 0n
+$(OUT)/registryDAO_storage.tz : c = 0n
+$(OUT)/registryDAO_storage.tz : d = 0n
+$(OUT)/registryDAO_storage.tz: src/**
+
+	# ============== Compiling RegistryDAO storage ============== #
+	mkdir -p $(OUT)
+	$(BUILD_STORAGE) --output-file $(OUT)/registryDAO_storage.tz src/registryDAO.mligo base_DAO_contract 'default_registry_DAO_full_storage(("$(admin_address)": address), ("$(token_address)": address), ${a}, $(b), $(s_max), $(c), $(d))'
+	# ============== Compilation successful ============== #
+	# See "$(OUT)/registryDAO_storage.tz" for compilation result #
+
 test: $(OUT)/baseDAO.tz
 	$(MAKE) -C .. test PACKAGE=baseDAO-ligo-meta \
     BASEDAO_LIGO_PATH=../$(OUT)/baseDAO.tz
```

Related to what @rinn7e was saying:
1. we won't need to manually set `REGISTRY_STORAGE_PATH` to run `make test` if we do it here
2. a `make test` after a `make clean` should work out of the box, but here we are missing the automatic generation of the storage

This will fix both things:
```suggestion
test: $(OUT)/baseDAO.tz $(OUT)/registryDAO_storage.tz
	$(MAKE) -C .. test PACKAGE=baseDAO-ligo-meta \
    BASEDAO_LIGO_PATH=../$(OUT)/baseDAO.tz \
    REGISTRY_STORAGE_PATH=../$(OUT)/registryDAO_storage.tz
```